config/testgrids/kubernetes/sig-node/config.yaml

```diff
@@ -17,6 +18,7 @@ dashboard_groups:
 
 
 dashboards:
+- name: sig-node-containerd-io
```

Comment:
Ooh, I can't wait to see if this goes to first in the list.

I want to suggest putting it in the same order as the other list, but I don't know how the inside of testgrid works.